akka-http-core/src/main/scala/akka/http/scaladsl/model/Multipart.scala

```diff
@@ -43,20 +53,48 @@ sealed trait Multipart {
         .flatMapConcat(ConstantFun.scalaIdentityFunction)
     HttpEntity.Chunked(mediaType withBoundary boundary withCharset charset, chunks)
   }
+
+  /** Java API */
```

Comment:
Did Akka's policies wrt to this change at some point? In the optimal case, there wouldn't be any JavaDoc for this Scala file and the methods would completely be hidden in ScalaDoc. From the Scala side these methods are just inevitable and useless boilerplate. Wasn't that your original reasoning why we had to add those comments?